js/event_target.ts

```diff
@@ -0,0 +1,49 @@
+// Copyright 2018 the Deno authors. All rights reserved. MIT license.
+import * as domTypes from "./dom_types";
+
+export class EventTarget implements domTypes.EventTarget {
+  public listeners: {
+    [type in string]: domTypes.EventListenerOrEventListenerObject[]
+  } = {};
+
+  public addEventListener(
+    type: string,
+    listener: domTypes.EventListenerOrEventListenerObject | null,
+    options?: boolean | domTypes.AddEventListenerOptions
+  ): void {
+    if (!(type in this.listeners)) {
+      this.listeners[type] = [];
+    }
+    if (listener !== null) {
+      this.listeners[type].push(listener);
+    }
+  }
+
+  public removeEventListener(
+    type: string,
+    callback: domTypes.EventListenerOrEventListenerObject | null,
+    options?: domTypes.EventListenerOptions | boolean
```

Comment:
Same as above, also, nit, the ordering is inconsistent.

Reply:
ditto above